src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala

```diff
@@ -2,14 +2,21 @@ package org.constellation
 
 import cats.effect.{ContextShift, IO}
 import org.constellation.infrastructure.snapshot.{SnapshotInfoLocalStorage, SnapshotLocalStorage}
+import org.constellation.serializer.KryoSerializer
+import cats.implicits._
+import org.scalatest.BeforeAndAfter
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
 import scala.concurrent.ExecutionContext
 
-class SnapshotHashIntegrityTest extends AnyFreeSpec with Matchers {
+class SnapshotHashIntegrityTest extends AnyFreeSpec with Matchers with BeforeAndAfter {
   implicit val cc: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
 
+  before {
+    KryoSerializer.init[IO].handleError(_ => Unit).unsafeRunSync()
```

Comment:
This line is duplicated many times. Maybe it's worth extracting this to a trait that test classes would extend? Or any other way of removing dupliation.

Reply:
KryoSerializer will be rewritten to `class` instead of `object` anyway so I would avoid spending time on clearing that repetition as we will get rid of that anyway.